docs/spec/spec.md

```diff
@@ -1608,6 +1609,45 @@ To resolve the value of a field, the `resolve()` method can be used. This requir
 public isolated function resolve(graphql:Field ‘field) returns anydata;
 ```
 
+### 8.6 Adding ErrorDetail to the Context
+
+To add an `ErrorDetail` to the context, the `addError()` method can be used. This method requires a `ErrorDetail` object as the parameter.
```

Comment:
I believe it's important to clarify how including an error in the context can affect the overall GraphQL response.

Reply:
Overall Graphql response will be affected as in this example.
https://github.com/ballerina-platform/module-ballerina-graphql/blob/master/docs/spec/spec.md#example-sample-response-for-error-returned-from-a-nullable-field
I'll update it.